README.CentOS7.md

```diff
@@ -0,0 +1,157 @@
+
+*Note:* LORIS developers (those NOT working with a .zip release codebase) should skip steps relating to hosting their database locally. Contact your system administrator for database credentials.
+
+```bash
+sudo yum install MariaDB-client MariaDB-server
+```
+
+By default in CentOS 7, the MariaDB version is 10.2
+Check what version you have installed by running:
+```
+mysql -V
+```
+
+Upgrade your MariaDB to version 10.3 (LORIS 21 does not support MariaDB 10.2.27)
+Then, 
+```bash
+sudo systemctl start mariadb 
+sudo systemctl enable mariadb
+sudo systemctl status mariadb
+```
+
+To finalise the MariaDB installation: 
+```bash
+sudo mysql_secure_installation
+```
+Then follow instructions to create a password for the root user.
+
+## NodeJS
+```bash
+sudo yum install nodejs
+```
+
+## PHP Composer
+```bash
+sudo curl -sS https://getcomposer.org/installer | php
+sudo mv composer.phar /usr/local/bin/composer
+```
+# Get the Code
+
+Download the latest release from the [releases page](https://github.com/aces/Loris/releases) to the home directory (~/), unzip it, and copy the contents to your project directory, `/var/www/loris` (we recommend naming your project directory `loris`, although you can use a different naming convention if you prefer). 
+```bash
+wget https://github.com/aces/Loris/archive/v$VERSION.tar.gz
+tar -zxf Loris-%VERSION%.tar.gz
+cp -r Loris-%VERSION%/ /var/www/loris
+```
+
+Alternatively the latest development branch can be obtained by forking the [LORIS repository](http://github.com/aces/Loris) for development purposes. We do not support unstable dev branches. 
+
+# Configure Apache
+
+A sample apache configuration file is in `docs/config/apache2-site`. 
+The install script will ask if you want to automatically create/install apache config files.
+It is recommended to perform this step **manually** --: copy our sample file to the apache configuration directory (`/etc/httpd/conf.d/`) and add the `.conf` file extension:
+
+```bash
+cd /var/www/loris
+cp docs/config/apache-site /etc/httpd/conf.d/apache-site.conf
+```
+
+Customize and Verify your settings: 
+* Paths and settings in `/etc/httpd/conf.d/apache-site.conf` should be populated appropriately for your server. Replace placeholders such as `%LORISROOT%` with `/var/www/loris`, `%PROJECTNAME%` with `loris`, `%LOGDIRECTORY%` with `/var/log/httpd/loris-error.log` 
+ * DocumentRoot should point to `/var/www/loris/htdocs`
+ * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
+
+Finally, restart apache:
+```bash
+sudo systemctl restart httpd
+sudo systemctl status httpd
+```
+# Install LORIS
+
+For the purpose of following LORIS conventions and easy understanding of all LORIS documentation, we recommend the following account names: 
+
+* `lorisadmin` unix user with sudo permission who will setup and manage the LORIS back-end
```

Comment:
If we recommend `lorisadmin` not to have sudo permission, this line should be modified

Reply:
Hi @PapillonMcGill  I think this inconsistency/irregularity has to be smoothed out via discussion per John's new issue #5297 on this topic - so we have clarity and consensus across the team. 
Definitely noted; at the same time, getting this PR merged quickly on `bugfix` for 21.0.3, to correct for the grosser inaccuracies in the major release documentation, would be ideal (whereas sudo is a clarification of our design/practices going forward).